build.sbt

```diff
@@ -38,6 +38,8 @@ libraryDependencies ++= {
     "com.softwaremill.sttp.tapir" %% "tapir-openapi-circe-yaml" % tapirVersion,
     "com.softwaremill.sttp.tapir" %% "tapir-swagger-ui-http4s"  % tapirVersion,
     "org.http4s"                  %% "http4s-blaze-server"      % http4sVersion,
+    "org.http4s"                  %% "http4s-dsl"               % http4sVersion,
+    "org.http4s"                  %% "http4s-server"            % http4sVersion,
```

Comment:
Do you know if we need this if we have `http4s-blaze-server`?

Reply:
Not sure about this, actually.